WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingViewModel.swift

```diff
@@ -27,25 +30,53 @@ final class BlazeAdDestinationSettingViewModel: ObservableObject {
         return String(format: Localization.finalDestination, buildFinalDestinationURL())
     }
 
+    // View model for the add parameter view.
+    var blazeAddParameterViewModel: BlazeAddParameterViewModel {
+
+        // The remaining characters to be used for validation in `BlazeAddParameterViewModel.
+        // If more than 1 parameter exist and user is adding a new character, the value should be reduced further by 1.
+        // This to take into account the need to add "&" separator character to the new parameter.
+        var adjustedRemainingCharacters = calculateRemainingCharacters()
+        if parameters.count > 1 && selectedParameter == nil {
+            adjustedRemainingCharacters = calculateRemainingCharacters() - 1
+        }
```

Comment:
Nit: for readability, you can use lazy variable:

```suggestion
        let adjustedRemainingCharacters: Int = {
            let remainingCharacters = calculateRemainingCharacters()
            if parameters.count > 1 && selectedParameter == nil {
                return remainingCharacters - 1
            }
            return remainingCharacters
        }()
```

Reply:
Thanks, this does read better to me now.